### DISCUSSION TITLE
Did DBeaver update the version to fix Eclipse bug 579335? Errors in importting long line into sql editor

### DISCUSSION THREAD

**Discussion by Rilfoyle** [dbeaver/dbeaver#20857] [Q&A] [answered]
I've met a problem when import sql script which is one line (about 10000 chars). DBeaver is constantly using more memory.
![image](https://github.com/dbeaver/dbeaver/assets/48911200/7d62b17a-3796-48a9-9d40-d98c15bc3b5c)
I found this bug caused by ECLIPSE' SWT, and it has been fixed. 
https://bugs.eclipse.org/bugs/show_bug.cgi?id=579335
https://git.eclipse.org/r/c/platform/eclipse.platform.swt/+/192078
I wonder has dbeaver update the swt to fix this bug? I'd like to know which version if fixed, thx.

**Comment by E1izabeth** [comment, marked as answer]
We update Eclipse as soon as the new release comes. You could try the latest DBeaver version - I couldn't reproduce it on DBeaver 23.1.4
